### PR TITLE
GH-2291: Add onboarding Step 0 ping

### PR DESCRIPTION
**Pull request by leuryr**
Adds a ping for the Welcome screen when setup is started through the addition of appropriate `dawn_setup_number`. I've added the value as a constant in `OnboardingConstants.js`, but I can remove that and just include the value directly if that's overkill.

---
* [x] Have you followed the guidelines in [CONTRIBUTING.md](https://github.com/ghostery/ghostery-extension/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

### CODE REVIEW

**Review by wlycdgr — COMMENTED** [review]
These changes look good but let's make sure that the ping is actually getting sent before approval

**Comment by leuryr** [comment]
Found the issue and pushed a fix. I'm able to see the correct values coming up in the Network tab now.

**Review by wlycdgr — APPROVED** [review]
Smoked tested. LGTM.

**Review by christophertino — APPROVED** [review]

**Pull request merged**
